static/absolute/js/searchPage.js

```diff
@@ -247,7 +240,7 @@ async function initMap() {
       });
     });
   } else {
-    //By default the map is centred around Los Angeles.
+    //Default Map is centred around Los Angeles.
```

Comment:
Nit spelling/capitalization:
- Default Map --> Default map
- centred --> centered

Reply:
British vs American spelling confusion here. hahaha

Reply:
I put map is caps for emphasis tho

Reply:
done